terraform/modules/ecs_module/security-groups.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_security_group" "aws-lb" {
+  name        = "${var.nginx_test_name}-load-balancer"
+  description = "${var.nginx_test_name}"
+  vpc_id      = aws_vpc.aws-vpc.id
+  ingress {
+    protocol    = "tcp"
+    from_port   = var.nginx_port
+    to_port     = var.nginx_port
+    cidr_blocks = var.app_sources_cidr
+  }
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  tags = {
+    Name = "${var.nginx_test_name}-alb-sg"
+  }
+}
+
+resource "aws_security_group" "aws-ecs-tasks" {
+  name        = "${var.nginx_test_name}-ecs-tasks"
+  description = "Allow inbound access from the ALB only"
+  vpc_id      = aws_vpc.aws-vpc.id
+
+  ingress {
+    protocol        = "tcp"
+    from_port       = var.nginx_port
+    to_port         = var.nginx_port
+    security_groups = [aws_security_group.aws-lb.id]
+  }
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
**[tfsec]** <[AVD-AWS-0104](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/ec2/no-public-egress-sgr/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Security group rule allows egress to multiple public internet addresses.